projects/observability/src/shared/components/topology/topology.component.ts

```diff
@@ -58,9 +59,18 @@ export class TopologyComponent implements OnChanges, OnDestroy {
   @Input()
   public shouldAutoZoomToFit?: boolean = false;
 
+  @Input()
+  public draggableNodes?: boolean = true;
```

Comment:
Nit: this sounds like it is storing the nodes which are draggable. Could we rename it to allowNodeDrag?

Reply:
Hmm, this was the property already present in the topology config, so I used the same name. do we change at both places or just input?

Reply:
It is fine if it is an existing term